compiler/rustc_const_eval/src/interpret/intrinsics.rs

```diff
@@ -485,6 +485,39 @@ impl<'tcx, M: Machine<'tcx>> InterpCx<'tcx, M> {
                 }
                 self.copy_op(&self.project_index(&input, index)?, dest)?;
             }
+            sym::simd_shuffle => {
```

Comment:
We have an existing implementation at https://github.com/rust-lang/miri/blob/cc27f3d6361342cb461cc45be52ec4e1b2f839d1/src/intrinsics/simd.rs#L658-L688, please just move that over. (It looks very similar though.)